From f8fe2f30e0958fc9f916d1757ca745d4b09c7102 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Caio=20Rom=C3=A3o?= <caioromao@gmail.com>
Date: Wed, 3 Jun 2009 17:44:07 -0300
Subject: [PATCH 01/10] encode_multipart_formdata fails during attach().

`bug_id` get's added to L as an integer during
encode_multipart_formdata() making it fail on CRLF.join(L). Casting
`bug_id` to string on attach() fixes this issue.
---
 bugz/bugzilla.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bugz/bugzilla.py b/bugz/bugzilla.py
index d894e45..df9e13c 100644
--- a/bugz/bugzilla.py
+++ b/bugz/bugzilla.py
@@ -779,7 +779,7 @@ class Bugz:
 			self.auth()
 
 		qparams = config.params['attach_post'].copy()
-		qparams['bugid'] = bugid
+		qparams['bugid'] = str(bugid)
 		qparams['description'] = title
 		qparams['comment'] = description
 		qparams['contenttypeentry'] = content_type
-- 
1.6.3.2

